src/timer/ExamTimerBlock.jsx

```diff
@@ -66,15 +66,15 @@ const ExamTimerBlock = injectIntl(({
           <div>
             <FormattedMessage
               id="exam.examTimer.text"
-              defaultMessage='You are taking "'
-            />
-            <Alert.Link href={attempt.exam_url_path}>
-              {attempt.exam_display_name}
-            </Alert.Link>
-            <FormattedMessage
-              id="exam.examTimer.text"
-              defaultMessage='" as {examType}. '
-              values={{ examType: attempt.exam_type }}
+              defaultMessage='You are taking "{examLink}" as {examType}'
```

Comment:
NIT: this is just missing the period and trailing space to separate out the 'Show More' link.

How it looks today "You are taking "My Exam" as a timed exam. _Show more_"

Reply:
Thanks a lot @Zacharis278 for the detailed review. I've added a spacer [`{' '}`](https://techstacker.com/how-to-add-white-space-between-elements-react-jsx/#add-space-with-curly-braces--quotes) outside the message and got back the `.` inside. This helps translators since spaces are invisible.